ibm/service/power/resource_ibm_pi_instance_test.go

```diff
@@ -44,21 +44,22 @@ func testAccCheckIBMPIInstanceConfig(name, instanceHealthStatus string) string {
 	  }
 	  resource "ibm_pi_instance" "power_instance" {
 		pi_memory             = "2"
-		pi_processors         = "0.25"
+		pi_processors         = "1"
 		pi_instance_name      = "%[2]s"
-		pi_proc_type          = "shared"
+		pi_proc_type          = "dedicated"
 		pi_image_id           = data.ibm_pi_image.power_image.id
 		pi_key_pair_name      = ibm_pi_key.key.key_id
-		pi_sys_type           = "s922"
+		pi_sys_type           = "e980"
 		pi_cloud_instance_id  = "%[1]s"
 		pi_storage_pool       = data.ibm_pi_image.power_image.storage_pool
 		pi_health_status      = "%[5]s"
 		pi_volume_ids         = [ibm_pi_volume.power_volume.volume_id]
 		pi_network {
 			network_id = data.ibm_pi_network.power_networks.id
 		}
+		pi_deployment_type          = "%[6]s"
 	  }
-	`, acc.Pi_cloud_instance_id, name, acc.Pi_image, acc.Pi_network_name, instanceHealthStatus)
+	`, acc.Pi_cloud_instance_id, name, acc.Pi_image, acc.Pi_network_name, instanceHealthStatus, acc.Pi_deployment_type)
```

Comment:
The default value for deployment type, terraform-test-power, will not work. I am not sure  pi_deployment_type can be set to "" but maybe that will work but if not you could make another dedicated test case for EPIC deployment types since this deployment type has other restrictions that the basic config does not have like it requires dedicated procs, tier1 storage and does not work on s922

Reply:
"" will work as already tested by her.
I asked her to integrate this with the existing test case to save on just adding new test func. This was before I came to know about the processors limitation.
Now, I will leave it up to you folks whether you want to have a new test case or use the existing one with some comment about the limitation to whoever will be using this env var.

Reply:
I told her I was OK to add a new test case unless you had a reason to not do that. So I am good with it. 
Bhagyashri, can you please update the description with the passing statement for the new test case.